docs/UserGuide.md

```diff
@@ -94,17 +86,20 @@ These are all the available commands in *Teletubbies*.
 To navigate to the section for each command, click on the hyperlinked commands in the left column. 
 The sections outline the detailed usage and examples for each of the command.
 
-Command | Format
+System commands | Format
 --------|------------------
 **[help](#viewing-help--help)** | `help`
 **[import](#importing-contacts--import)** | `import`
-**[merge](#merging-contacts-merge)** | `merge`
+**[merge](#merging-contacts--merge)** | `merge`
 **[export](#exporting-contacts-export)** | `export [-t TAGS]…​`
 **[profile](#setting-your-profile-profile)** | `profile -n NAME -role ROLE`
 **[history](#listing-your-previous-commands--history)** | `history`
```

Comment:
Should be `--` for Lines 94, 95, 103, 116, 117 as well?

Reply:
Only the hyperlink for merge was broken

Reply:
Fixed the inconsistency between the use of "-" and "--", some commands had extra whitespace. Have removed the whitespace to standardize all commands.